website/docs/d/nas_access_rules.html.markdown

```diff
@@ -37,6 +39,8 @@ The following arguments are supported:
 * `source_cidr_ip` - (Optional) Filter results by a specific SourceCidrIp. 
 * `user_access` - (Optional) Filter results by a specific UserAccess. 
 * `rw_access` - (Optional) Filter results by a specific RWAccess. 
+* `ipv6_source_cidr_ip` - (Optional,Available in v1.189.0+) The IPv6 address or CIDR block of the authorized object.
+                                                            You must set this parameter to an IPv6 address or CIDR block.
```

Comment:
This parameter cannot be set at the same time as the `source_cidr_ip` parameter.